app/src/main/java/ch/epfl/sdp/drone3d/map/gps/LocationComponentManager.kt

```diff
@@ -66,7 +66,10 @@ class LocationComponentManager(private val activity: Activity, val mapboxMap: Ma
             }
 
             if (locationService.isLocationEnabled()) {
-                MapboxUtility.zoomOnCoordinate(locationService.getCurrentLocation()!!, mapboxMap)
+                if(locationService.getCurrentLocation() != null){
```

Comment:
Could be done in 1 if but I think Jose corrected this issue, it might be worth it to update this branch with the correction.